tests/lib/tools/tests.pkgs.apt.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+remap_one() {
+    case "$1" in
+        man)
+            if [ "$(cat /etc/os-release && echo "$ID")" = debian ]; then
```

Comment:
This doesn't seem to be present in the old (removed) code in this PR, is it needed given that base remapping on SPREAD_SYSTEM?

Reply:
Perhaps I made a mistake but the 14.04 code was doing one remapping and falling back to debian which remapped one more thing (man).

Reply:
This is from https://github.com/snapcore/snapd/pull/9479/files#diff-6eea9b76e2bf27f60519f7b5c98333c9R6 - debian re-mapped `man` to `man-db`